src/components/Datatable/Datatable.tsx

```diff
@@ -8,13 +8,15 @@ import { getBorderRadius, getColor } from '../../utils/helpers';
 import { FlexContainer } from '../FlexContainer';
 import { ActionPropType } from './types/Action.types';
 import Table from './Table/Table';
-import { BatchModule } from './BatchModule';
 import DatatableContext from './DatatableContext';
 import {
   ControlsConfig,
   DatatableProps,
   ExtendedTableConfig,
 } from './Datatable.types';
+import { ControlModule } from './ControlModule';
+import { BatchModule } from './BatchModule';
+import { mockControlModuleProps } from './mocks/controlModule';
```

Comment:
Since Datatable is already released there should not be any mocks in the components.

Reply:
mocks removed